modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/modules/frontend-js-web.d.ts

```diff
@@ -12,20 +12,6 @@
  * details.
```

Comment:
Ah I see. This is a root issue of SF removing dependencies when it shouldn't be then. We can just allow this for now and hopefully we will provide support from the infra side so that you don't need to do this.